src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,7 +44,9 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(arrElement -> arrElement.equals(element))
+                .findAny().orElseThrow(NoSuchElementException::new);
```

Comment:
NoSuchElementException is thrown by default. You do not need to write it in orElseThrow parameter